src/main/java/gregtech/api/metatileentity/MetaTileEntity.java

```diff
@@ -1261,6 +1291,18 @@ public void onAttached(Object... data) {
      * tile entity will still get getDrops called after this, if player broke block
      */
     public void onRemoval() {
+        if (getWorld().isRemote) {
+            GTSoundManager.stopTileSound(getPos());
+        }
+    }
+
+    @SideOnly(Side.CLIENT)
+    public SoundEvent getSound() {
+        return null;
```

Comment:
Why doesn't this just return `getRecipeMap().getSound()`? This would prevent some overrides in subclasses most likely.